cmd/version.go

```diff
@@ -18,6 +18,7 @@ package cmd
 
```

Comment:
Hi @migueleliasweb and @dmvolod 

Could we ensure that this one is rebased with master?
Also, is this one good to fly or what would be missing?
Is possible ensure that we have in the description an example of test executed for we ensure that all is fine and make easier the review/validation?

Reply:
Sorry all, I've got stuck on a crazy hectic week and a bit at work. Once this is off by back, I'll come back to this if that's okay. :pray: I didn't forget this PR!